docs/monthly-meeting/2022-12.md

```diff
@@ -0,0 +1,84 @@
+    * -> No JS PR: https://github.com/python/devguide/pull/997
+
+* https://packaging.python.org/en/latest/glossary/
+  * @CAM-Gerlach to improve the glossary. It can then be linked or copied to CPython docs.
+
+### PEP Workflow
+
+* [PEP 1 rewrite?](https://discuss.python.org/t/21068/26)
+  * It was generally agreed that PEP 1 (and PEP 12) could use significant re-organization and trimming.
+  * Petr suggests moving the author-facing help out of PEP 1/to the devguide
+  * CAM mentions [python/peps#2337](https://github.com/python/peps/issues/2337) which proposes referencing the Sphinx reST primer instead of duplicating information there in PEP 12 and consolidating normative process guidance in PEP 1 and author-facing info in PEP 12/the devguide
+  * @CAM-Gerlach will open a PR implementing that first step, linking the Sphinx reST primer in PEP 12 instead of duplicating that information
+  * CAM also mentions the devguide's reST primer is essentially just a less-up-to-date copy of the Sphinx one; it should be revised to link to it and focus on CPython-specific guidance. This issue was already open as [python/docs-community#57](https://github.com/python/docs-community/issues/57), which was moved to [python/devguide#1000](https://github.com/python/devguide/issues/1000), where @CAM-Gerlach will work on it
+* Final PEPs and the canonical docs banner link
+  * [python/peps#2719](https://github.com/python/peps/issues/2719) will improve the existing banners directing users to the canonical docs for Final PEPs added in [python/peps#2702](https://github.com/python/peps/pull/2702) to be sticky at the top; @CAM-Gerlach to submit PR
+  * We can start going through Final PEPs and systemically marking them with the new banner as appropriate
+  * @CAM-Gerlach will submit PRs for [python/peps#2872](https://github.com/python/peps/issues/2872) to mark old implemented Accepted PEPs as Final, and will add banner in the process
+  * We might need a checklist for marking PEPs Final
+    * e.g.: is it documented outside the PEP? Add a note?
+    * @CAM-Gerlach will open an issue to discuss adding checklists for new Approved/Rejected and Final PEPs
+* Status for PEP 659?
+  * @CAM-Gerlach will open issue proposing a list of obselete PEPs to the SC to be formally rejected
+  * We may need an obselete status?
+
+### Improve SEO for CPython docs
+
+* (Hugo) OpenGraph metadata PRs:
+
+  * devguide: [merged](https://github.com/python/devguide/pull/953), increases Google's SEO score [91->100](https://github.com/python/devguide/pull/953#issuecomment-1332183148)
+  * CPython: [ready for review](https://github.com/python/cpython/pull/99931), increases Google's SEO score [~66->91](https://github.com/python/cpython/pull/99931) -> approved during and merged after the meeting :rocket:
+  * Followup: tell Marc-André about use of logo without font
```

Comment:
This is now done, we removed the text and left the logo, so the font is no longer a problem.